apis/Google.Cloud.Diagnostics.AspNetCore/Google.Cloud.Diagnostics.AspNetCore/Logging/LoggerOptions.cs

```diff
@@ -56,21 +67,27 @@ public sealed class LoggerOptions
         /// Create a new instance of <see cref="LoggerOptions"/>.
         /// </summary>
         /// <param name="logLevel">Optional, the minimum log level.  Defaults to <see cref="LogLevel.Information"/></param>
+        /// <param name="logName">Optional, the name of the log.  Defaults to 'aspnetcore'.</param>
```

Comment:
If we're keeping this as the first parameter, let's move the docs for it above `logLevel`. But perhaps it should be the second parameter anyway? What are users more likely to want to change - the name or the level?

Reply:
Log level is more likely to change.  Updated.

Reply:
Sounds good.  This will be put in the next major release which is 3.0

Reply:
Sorry, had forgotten we were already at 2.x. Glad you caught my meaning :)